timed_proc_macros/src/lib.rs

```diff
@@ -119,18 +119,19 @@ pub fn timed(args: TokenStream, input: TokenStream) -> TokenStream {
         block: body,
         ..
     } = &function;
-
-    let name = &function.sig.ident;
+    let name = format!("{}::{}", std::module_path!(), &function.sig.ident);
```

Comment:
Just tested this out, the output of:
```
cargo run --package tests --example demo_tracing
```
is:
```
[
    { "pid": 0, "ts": 1603479130104696,  "ph": "Start", "name": "timed_proc_macros::foo" },
    { "pid": 0, "ts": 1603479130104699,  "ph": "Start", "name": "timed_proc_macros::bar" },
    { "pid": 0, "ts": 1603479130104700,  "ph": "Start", "name": "timed_proc_macros::baz" },
    { "pid": 0, "ts": 1603479130104752,  "ph": "Finish", "name": "timed_proc_macros::baz" },
    { "pid": 0, "ts": 1603479130104756,  "ph": "Finish", "name": "timed_proc_macros::bar" },
    { "pid": 0, "ts": 1603479130104760,  "ph": "Start", "name": "timed_proc_macros::baz" },
    { "pid": 0, "ts": 1603479130104765,  "ph": "Finish", "name": "timed_proc_macros::baz" },
    { "pid": 0, "ts": 1603479130104768,  "ph": "Finish", "name": "timed_proc_macros::foo" }
]
```

It should in fact be `demo_tracing::foo`, not `timed_proc_macros::foo`

You should add this inside: `quote!`, I just tried:

```rust
let result = quote! {
        #(#attrs)*
        #vis #sig {
           #tracing_begin
           let start = std::time::Instant::now();
           let res = { #body };
           let elapsed = start.elapsed();
           let path = std::module_path!();
           println!("Module path {:?}", path);
           #print_duration
           #tracing_end
           res
        }
    };
```

prints:
```
Module path "demo_tracing"
```